docker/dockerfiles/fe/fe_entrypoint.sh

```diff
@@ -71,12 +71,34 @@ collect_env_info()
     fi
 }
 
+# When StarRocks FE is starting up, mysql client may fail to connect to the FE instance.
+# So, if the error is "Can't connect to MySQL server", we can safely ignore.
+execute_sql_command()
+{
+    local svc=$1
+    local sql=$2
+    output=$(timeout 15 mysql --connect-timeout 2 -h $svc -P $QUERY_PORT -u root --skip-column-names --batch -e "$sql" 2>&1)
+    if [[ $? -eq 0 ]]; then
+      # If the command is successful, log the output to STDOUT
+      echo "$output"
+      return 0
+    fi
+    # check if error message contains "Can't connect to MySQL server"
+    if [[ "$output" == *"Can't connect to MySQL server"* ]]; then
+      # If the error is "Can't connect to MySQL server", we can ignore it.
```

Comment:
This may cause the initial FE node starts and waiting for 30 seconds with no output, neither not alive.

Take a look at https://github.com/StarRocks/starrocks/blob/main/docker/dockerfiles/allin1/services/director/run.sh#L107, if possible we can unify the check between fe entrypoint and allin1 entrypoint.

Reply:
Ok, I will consider to use it. 
if this case happens 
```
This may cause the initial FE node starts and waiting for 30 seconds with no output, neither not alive.
```
the mysql command will return 1 and with an error message.